ouroboros-consensus-cardano/changelog.d/20231002_154149_armandoifsantos_fix_peerSharing_handshake.md

```diff
@@ -0,0 +1,20 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Patch
+
+- A bullet item for the Patch category.
+
+-->
+### Non-Breaking
+
+- Changes `NoPeerSharing` to `PeerSharingDisabled` ([#4644](https://github.com/input-output-hk/ouroboros-network/pull/4644))
+
+### Breaking
+
+- Adds `NodeToNodeVersion` to the arguments of `immDBServer` function.
```

Comment:
`immDBServer` is not technically part of our interface, since it's in an "unstable-*" package. I therefore think this change should instead be at most a Patch-level item, or even just not mentioned in the changelog.

Reply:
Got it, thanks!